backend/attendance-service/src/main/java/org/egov/web/models/AttendanceRegisterRequest.java

```diff
@@ -21,7 +23,7 @@ public class AttendanceRegisterRequest {
     private RequestInfo requestInfo = null;
 
     @JsonProperty("attendanceRegister")
-    private AttendanceRegister attendanceRegister = null;
+    private List<AttendanceRegister> attendanceRegister = null;
```

Comment:
Have you made the corresponding change in the API contract (.yaml file)?

Reply:
Yes, corresponding change has been made in API contract